content/post/2019-11-26-pkgsearch.Rmarkdown

```diff
@@ -0,0 +1,137 @@
+---
+title: Search and discover CRAN packages with pkgsearch!
+date: '2019-10-26'
+slug: pkgsearch
+tags:
+  - pkgsearch
+---
+
+Version 3.0.1 of the `pkgsearch` package has been [released on CRAN](https://cran.r-project.org/package=pkgsearch)! :tada: `pkgsearch` munges CRAN metadata and lets you
+access it through several lenses: **search packages by keyword, popularity, recent activity, package name and more.** 
+
+Get `pkgsearch`'s latest version from 
+a [safe](https://rud.is/b/2019/03/03/cran-mirror-security/) CRAN mirror near
+you:
+
+```r
+install.packages("pkgsearch")
+```
+
+This release, compared to `pkgsearch` 2.0.0, brings 
+
+:mag_right: Better package search functionality
+
+*  New RStudio addin to search for packages in a GUI: `pkg_search_addin()`.
+
+* New function `advanced_search()` for more search flexibility.
+
+:mag_right: New functionality adopted from former crandb
+
+* `cran_package()`, `cran_packages()` and `cran_package_history()` functions to query metadata about certain packages.
+    
+* `cran_events()` function to list recent CRAN events, new, updated or archived packages.
+    
+* `cran_top_downloaded()` function to query packages with the most downloads.
+    
+* `cran_trending()` function to return the trending CRAN packages.
+
+That's... a lot! Let's have some fun with the package.
+
+## Explore packages in a GUI
+
+You can use `pkgsearch` with an interface. See more in the video below. You can also launch the interface in a browser without RStudio, [refer to the addin docs](https://r-hub.github.io/pkgsearch/reference/pkg_search_addin.html).
+
+```{r addin, echo = FALSE}
+blogdown::shortcode("vimeo", "375618736")
+```
+
+## Other use cases
+
+The `pkgsearch` package is the answer to many questions you might have about CRAN metadata, thanks to its exposing it through different endpoints.
```

Comment:
"about CRAN packages"? also, I think we can just talk about the functions in the package, and not endpoints. No need to go into web apis here.